kiosk/settings/prod.py

```diff
@@ -20,16 +20,22 @@
 
 DEBUG = False # bool(os.getenv('DJANGO_DEBUG', ''))
 
-# Set to your Domain here (eg. 'yourwebsite.com')
-ALLOWED_HOSTS = ['0.0.0.0', 'localhost', '127.0.0.1', 'greetingkiosk.herokuapp.com']
+if DEBUG:
+    ALLOWED_HOSTS = ['*']
+else:
+    # Set to your Domain here (eg. 'yourwebsite.com')
+    ALLOWED_HOSTS = ['0.0.0.0', 'localhost', '127.0.0.1', 'greetingkiosk.herokuapp.com']
 
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql',
-        'NAME': 'postgres',
-        'USER': 'postgres',
-        'PASSWORD': 'postgres',
-        'HOST': 'db',
+        'NAME': 'greeting_kiosk',
+        'USER': 'aisvisioner',
+        'PASSWORD': 'aisvisioner',
+        'HOST': 'localhost',
         'PORT': 5432,
     },
-}
\ No newline at end of file
+}
+
+db_from_env = dj_database_url.config(conn_max_age=600)
```

Comment:
This simple Django utility allows you to utilize the 12factor inspired DATABASE_URL environment variable to configure your Django application.